src/library_ccall.js

```diff
@@ -31,6 +31,10 @@ mergeInto(LibraryManager.library, {
 #endif
       'string': (str) => {
         var ret = 0;
+         if (typeof str !== 'string')
+         {
+         assert(typeof str == 'string', 'Argument must be a string');
```

Comment:
You don't need the surrounding `if` block since the assert does that part for you.

And can you add `#if ASSERTIONS` and `#endif` around the assert line?

Reply:
sure

Reply:
I don't need to close the PR to make change to it, you can just push a new version.

Reply:
I am sorry I don't understand what you mean by it